execution/executor/src/block_executor/mod.rs

```diff
@@ -176,6 +180,7 @@ where
 struct BlockExecutorInner<V> {
     db: DbReaderWriter,
     block_tree: BlockTree,
+    global_cache_manager: GlobalCacheManager,
```

Comment:
and this is the actual place. 
after my PR this now has    ` block_executor: V,`, so having GlobalCacheManager inside V should be enough

Reply:
Yes! Let me rebase now that your PRs are landed. We still need APIs to mark/check caches here (we need to check block_hash/version) but the good thing is that there is no need to propagate the manager everywhere 👍